submissions/evgeniy24/friends_app/app.js

```diff
@@ -0,0 +1,248 @@
+const REQUEST_URL = 'https://randomuser.me/api/?results=20',
+    HEADER = document.querySelector('.header-name'),
+    FRIENDS_LIST_WRAP = document.querySelector('.friendsList-wrap'),
+    MENU_WRAP = document.querySelector('.menu-wrap'),
+    SEARCH_FIELD = document.querySelector('#search'),
+    SORTING_MENU = document.querySelector('.sorting-menu'),
+    FILTER_MENU = document.querySelector('.filter-menu'),
+    MOBILE_MENU_BUTTON = document.querySelector('.button-container');
+let friendsList = [],
+    searchResultList,
+    sortedList,
+    errorButton,
+    genderNeutralList, filteredByGenderList,
+    scrollDepth,
+    lettersArr = HEADER.innerHTML.split('');
```

Comment:
Ehm... what? What for? Just wrap every letter in span in HTML in a first place.

Reply:
i removed this solution